### PR TITLE
Add stats to users (and attendees) page

### LINKED ISSUE #222: All Users statistics information
## Is your feature request related to a problem?
When looking at the "[All] Users" page it does not show the status of each registered user. There is a similar set of stastics on the "Who's Coming" page for registered users.

## Describe the feature you'd like to see on the Go Congress Website.
On the "Who's Coming" page there is a line showing [total attendees, # dan players, # kyu players, # pros]. When looking at the "Users" page as an admin there is not a way to determine the number of users registered. It would be beneficial to see [Total Users, # attending, # not fully registered (accounts but not registered, [count of # Attendees column == 0]) ] 

## Notes
Add any other information or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by neagle**
2021-06-20 at 4 21 PM

Closes #222

### CODE REVIEW

**Comment by neagle** [comment]
@vash3g Could you let me know if this meets the needs of your issue? Thanks!

**Comment by vash3g** [comment]
If the numbers are above 10 do they show the numerals instead of the words?

**Comment by neagle** [comment]
Yup! Just a style convention.

--
Nate Eagle

File: Gemfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ gem 'devise'
 gem 'exception_notification'
 gem 'ffi'
 gem 'haml'
+gem 'humanize'
```
Comment (jaredbeck) [inline Gemfile:12]: Instead of adding a dependency, can we use something built into Rails for this? Like `number_to_human`?

**Review by jaredbeck — APPROVED** [review]
If we can avoid adding a dependency, that'd be great. Otherwise, LGTM.

**Comment by vash3g** [comment]
LGTM.

File: Gemfile (hunk 1 above)
Comment (neagle) [reply Gemfile:12]: `number_to_human` does something slightly different: it turns numbers like 12345 into "12.3 Thousand".

Humanize, on the other hand, turns numbers like 25 into "twenty-five". Publications commonly have style requirements that call for writing out numerals less than 100 or less than 10, and I've always thought it's a really nice touch for a website to take enough care to reproduce that.

**Pull request merged**